tomcat-verrazzano/workshops/freetier/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-    "workshoptitle": "Helidon and Verrazzano Fundamentals",
+    "workshoptitle": "Deploy Tomcat application docker image to Oracle Kubernetes Cluster",
```

Comment:
@pandey-ankit The title in WMS should match the title in the manifest.json file. Could you update the finalized title in WMS?